Documentation: Kedro-Viz React component usage (#1954)
kedro-org/kedro-viz: README.md

import KedroViz from '@quantumblack/kedro-viz';
import '@quantumblack/kedro-viz/lib/styles/styles.min.css';

> const MyApp = () => <div style={{height: `100vh`}}><KedroViz data={json} /></div>;

Reviewer: can we add options here. thanks

Author: @rashidakanchwala like this?
```
<div style={{height: `100vh`}}>
  <KedroViz
      data={json}
      options={
        display: {
          globalToolbar: true,
          sidebar: true,
          metadataPanel: true,
          miniMap: true,
          expandAllPipelines: false,
        }
        visible: {
          exportBtn: true,
          labelBtn: true,
          layerBtn: true,
          pipelineBtn: true,
          sidebar: true,        
        }
        theme: "dark",
        tag: {
          enabled: {companies: true}
        },
        nodeType: {
          disabled: {parameters: true}
        }
      }
  />
 </div>
```

Reviewer: since we go in details later maybe just 
```
<div style={{height: `100vh`}}>
  <KedroViz
      data={json}
      options={/* Options to configure Kedro Viz */}
  />
</div>
```

We also recommend wrapping the `Kedro-Viz` component with a parent HTML/JSX element that has a specified height (as seen in the above example) in order for Kedro-Viz to be styled properly.

> For more information on how to use Kedro-Viz as a React component, please follow this [guide](https://www.npmjs.com/package/@quantumblack/kedro-viz)

Reviewer 2: ```suggestion
For more information on how to use Kedro-Viz as a React component, follow this [guide](https://www.npmjs.com/package/@quantumblack/kedro-viz)
```

> For more information on how to use kedro as React component with all possible props please refer to [Kedro-Viz on npm](https://www.npmjs.com/package/@quantumblack/kedro-viz)

Reviewer 3: ```suggestion
For more information on how to use Kedro-Viz as a React component, please follow this [guide](https://www.npmjs.com/package/@quantumblack/kedro-viz)
```
